src/components/common/Button/Button.tsx

```diff
@@ -26,18 +26,16 @@ const shadeColor = (color, percent) => {
 const colorUnderButtons = (hexColor) => {
   const result = /^#?([a-f\d]{2})([a-f\d]{2})([a-f\d]{2})$/i.exec(hexColor);
   return result
-    ? `rgba(${parseInt(result[1], 16) - 102},${parseInt(result[2], 16) - 55},${
-        parseInt(result[3], 16) - 14
-      },1)`
+    ? `rgba(${parseInt(result[1], 16) - 102},${parseInt(result[2], 16) - 55},${parseInt(result[3], 16) - 14
+    },1)`
     : null;
 };
 
 const onHoverBackgroundColor = (hexColor) => {
```

Comment:
Can we have types for the parameters, hexColor

Reply:
There is currently no way to validate hexColor values, is it okay if I use string type?

Reply:
I wish typescript had a hexcolor type but yeah I think that's fine to limit it to string only!